src/pages/index.tsx

```diff
@@ -1,9 +1,10 @@
+import backgroundTall from '@assets/backgroundTall.jpg';
```

Comment:
i noticed you have import @assets or @/assets. Should use just one way for consistency otherwise it is confusing